src/loaders/LoaderUtils.js

```diff
@@ -37,8 +38,27 @@ var LoaderUtils = {
 
 		return url.substr( 0, index + 1 );
 
+	},
+
+	splitByWhitespace: function ( text ) {
+
+		return ( isSafari() ) ? text.trim().split( /\s+/ ) : text.replace( /\n/g, " " ).
+			replace( /\r/g, " " ).
+			replace( /\t/g, " " ).
+			replace( /  +/g, " " ). // double space
+			trim().
+			split( " " );
+
 	}
 
 };
 
+function isSafari() {
```

Comment:
Please remove this function. We need a solution that is appropriate for all browsers.